mftf/2.3/configuration.md

```diff
@@ -163,22 +166,32 @@ Example:
 MAGENTO_RESTAPI_SERVER_PORT=5000
 ```
 
-### TESTS
+### *_BP
 
-Overrides base paths for the framework.
+Settings to override base paths for the framework.
 You can use it when the MFTF is applied as a separate tool.
-For example, when you need to place the MFTF and the Magento code base in separate projects.
+For example, when you need to place the MFTF and the Magento codebase in separate projects.
 
 ```conf
+MAGENTO_BP
 TESTS_BP
 FW_BP
 TESTS_MODULES_PATH
 ```
 
+#### MAGENTO_BP
+
+The path to a local Magento codebase.
+It enables the [`bin/mftf generate`][mftf generate] commands to parse all modules at `app/code` and `vendor` of the provided Magento codebase.
```

Comment:
It also enables the `bin/mftf run` commands to parse and run the tests.

Reply:
will change to: 
It enables the [`bin/mftf`][mftf] commands such as `run` and `generate` to parse all modules of the Magento codebase for MFTF tests.